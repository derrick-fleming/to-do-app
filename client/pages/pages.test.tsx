import Home from '../pages/home';
import { Provider } from 'react-redux';
import store from '../redux/generate-store';
import React from 'react';
import renderer from 'react-test-renderer';

it('renders Home correctly', () => {
  const tree = renderer
    .create(<Provider store={store}><Home /></Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
