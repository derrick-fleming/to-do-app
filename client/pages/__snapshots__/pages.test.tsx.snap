// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home correctly 1`] = `
<div
  className="container"
>
  <h1
    className="text-center mt-4"
  >
    My Todo List
  </h1>
  <div
    className="justify-content-center row"
  >
    <div
      className="col-md-8 col-11"
    >
      <form
        className=""
        onSubmit={[Function]}
      >
        <label
          className="fs-4 form-label"
          htmlFor="todo"
        >
          Write your Task
        </label>
        <input
          className="my-2 form-control"
          id="todo"
          onChange={[Function]}
          placeholder="Enter task"
          type="text"
          value=""
        />
        <div
          className="text-end"
        >
          <button
            className="fs-4 btn btn-primary"
            disabled={false}
            type="submit"
          >
            Submit
          </button>
        </div>
      </form>
    </div>
  </div>
  <div
    className="justify-content-center row"
  >
    <div
      className="col-md-8 col-11"
    >
      <div>
        <h6
          className="inline-block"
        >
          Filter Results
        </h6>
        <button
          className="btn-link"
          id="complete"
          onClick={[Function]}
        >
          Complete
        </button>
        <button
          className="btn-link"
          id="incomplete"
          onClick={[Function]}
        >
          Incomplete
        </button>
      </div>
    </div>
    <div
      className="col-md-8 col-11"
    >
      <ul>
        Loading...
      </ul>
    </div>
  </div>
</div>
`;
